src/lib/evm-toolkit/constants/networks.constant.ts

```diff
@@ -7,7 +7,7 @@ export const NETWORKS: INetwork[] = [
     "apiEndpoint": "https://evm-toolkit-api.testnet.shimmer.network",
     "faucetEndpoint": "https://faucet.testnet.shimmer.network",
     "chainAddress": "rms1prwgvvw472spqusqeufvlmp8xdpyxtrnmvt26jnuk6sxdcq2hk8scku26h7",
```

Comment:
the chainAddress also changed to: `rms1pr75wa5xuepg2hew44vnr28wz5h6n6x99zptk2g68sp2wuu2karywgrztx3`